mos-platform/atari8-stdcart/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+platform(atari8-stdcart COMPLETE PARENT atari8)
+
+if(NOT CMAKE_CROSSCOMPILING)
+  return()
+endif()
+
+install(FILES
+  cart-rom-fixed.ld
+  link.ld
+  stdcart.ld
+TYPE LIB)
+
```

Comment:
Since this is a ROM target, you'll need some code to copy the `.data` and `.zp.data` segment initializers from ROM to RAM when the program starts. This is provided via a libcrt0.a library:

```
add_platform_library(atari8-stdcart-crt0)
merge_libraries(atari8-stdcart-crt0 common-copy-data)
```